extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -102,6 +131,26 @@
          */
         @ConfigItem(defaultValue = "/group")
         String groupPath;
+
+        /**
+         * The path where Health UI is available.
+         * The value `/` is not allowed as it blocks the application from serving anything else.
+         */
+        @ConfigItem(defaultValue = "/health-ui")
+        String rootPathUi;
```

Comment:
Better move these properties to a `ui` subgroup

Reply:
@gastaldi - do you have an example of this ? I am not sure how/what to do here ? (I just copied from other with UI)

Reply:
just create a `ui` `@ConfigGroup` and a nested class with the UI elements.

Reply:
Thanks @gsmet !

Reply:
@gsmet  - This will change the config property right ? If so, probably we need to do this in the other UI services to be consistent ?

Reply:
Yes, the config property should be: `quarkus.smallrye-health.ui.root-path`. About the other UI services that would be good to do too

Reply:
Cool. Changes done, building local, then I'll push

Reply:
Done :)